libs/ledger/src/dag/dag_hash.cpp

```diff
@@ -0,0 +1,93 @@
+//------------------------------------------------------------------------------
+//
+//   Copyright 2018-2019 Fetch.AI Limited
+//
+//   Licensed under the Apache License, Version 2.0 (the "License");
+//   you may not use this file except in compliance with the License.
+//   You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+//   Unless required by applicable law or agreed to in writing, software
+//   distributed under the License is distributed on an "AS IS" BASIS,
+//   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//   See the License for the specific language governing permissions and
+//   limitations under the License.
+//
+//------------------------------------------------------------------------------
+
+#include "ledger/dag/dag_hash.hpp"
+
+using namespace fetch::ledger;
+
+inline char DAGHashTypeToChar(DAGHash::Type const &t)
```

Comment:
No need for `inline` here. Please wrap in anonymous namespace as well

Reply:
done